org.eclipse.scout.rt.dataobject.test/src/main/java/org/eclipse/scout/rt/dataobject/testing/signature/DataObjectSignatureGenerator.java

```diff
@@ -410,6 +409,13 @@ protected String processWildcardType(WildcardType wildcardType, Class<? extends
   }
 
   protected String processClass(Class<?> clazz, Class<? extends IDoEntity> containingEntityClass, String containingAttributeName) {
+    for (IDataObjectSignatureTestCustomizer c : m_customizers) {
+      String errorMsg = c.validateAttributeType(containingAttributeName, clazz, containingEntityClass);
```

Comment:
Change attribute order in IDataObjectTestCustomizer to the one used within processClass? clazz, containingEntityClass, containingAttributeName